.gitignore

```diff
@@ -0,0 +1,10 @@
+### Sass ###
+.sass-cache/
+.scss-cache/
+*.css.map
+*.sass.map
+*.scss.map
+*.css
+
+## System ##
+.DS_Store
```

Comment:
To avoid the little arrow to show we use to leave a empty line at each of the files :)